xdist/remote.py

```diff
@@ -251,17 +251,12 @@ def remote_initconfig(option_dict, args):
 
 
 if __name__ == "__channelexec__":
+    import py
+
     channel = channel  # noqa
     workerinput, args, option_dict = channel.receive()
```

Comment:
we should put a flag wrt importing in here so we pass in the input, the args, the options, and a flag on whether to sort out the pythonpath

Reply:
Good idea, done